### PR TITLE
[Setup Pipenv] Pipfile file + update dev doc

### LINKED ISSUE #40: Support new bee onboarding
I am trying to play around with this repo locally. However, I am missing several dependencies as I am not a major Python player.

Could u help to list all deps used? I can help with a virtual env for local dev testing.

### PR DESCRIPTION
**Pull request by Czhang0727**
Provides necessary dependencies version lock to avoid leak with Python 3.8 standard.
README.md is also updated.
Fixes #40

### CODE REVIEW

**Comment by Czhang0727** [comment]
TODO: Need to split dev / prod packages to reduce bundle size.

@andysim3d  Have a nice Thanks Giving buddy 👍

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pytest = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.8"
```
Comment (andysim3d) [inline Pipfile]: ^3.7

Comment (Czhang0727) [reply Pipfile]: not sure this will work:
https://docs.pipenv.org/advanced/

Maybe >=3.7?

Comment (Czhang0727) [reply Pipfile]: Per doc, we should target a single version of Python, set to 3.7 for version support.

**Comment by Czhang0727** [comment]
request for merge?

**Review by andysim3d — APPROVED** [review]

**Pull request merged**